14_system-prep-for-build.Rmd

```diff
@@ -1,9 +1,66 @@
 # Setup an R dev environment
 
-System prep necessary to install R packages from source.
+There are two different types of R packages you can install, **binary**
+packages and **source** packages. When you install released packages from CRAN
+using `install.packages()` you are installing **binary** packages. When you
```

Comment:
Do you think we need any sort of caveat / fine print re: macOS + Windows vs Linux?

Reply:
For macOS, you should highlight that binary is preferable in almost all cases, because if the source will not successfully install, it is very troublesome to fix usually. I've only migrated my R environment to Ubuntu recently, so _cum grano salis_ but it appears that by default, at least, packages are installed by compiling source, and so far I've had no failures with the large number of packages that I have ported over. Mercifully, I haven't had occasion to work in Windows.

Reply:
I deliberately left it out because there are so few linux users compared to the other two OSs.

Reply:
Yeah, all of our data bears this out. And those few linux users know all too well that they don't get CRAN binaries 😁